test/unit/worldwide_ab_test_helper_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class WorldwideAbHelperTest < ActiveSupport::TestCase
+  def subject
+    WorldwideAbTestHelper.new
+  end
+
+  test "content returns the parsed file " do
+    result = subject.content
+    assert result[:sample].present?
```

Comment:
I ummed and arred a bit over that myself. There's probably value in testing this over the actual content file as it will go some way to verifying the contents of the file is parseable etc. That said I also agree with your points. I've added another commit with an inline fixture. See what you think and we can decide whether to keep it in.